API-Samples/Tutorial/markdown/10-init_render_pass.md

```diff
@@ -105,10 +111,28 @@ optimal layout so that the GPU can read it more efficiently.
 Conversions from one layout to another are called layout transitions
 in Vulkan.
 You have control over these transitions and can invoke them
-with an image memory barrier command or as a subpass dependency in a render
-pass.
-In this case, the sample code uses image memory barrier commands, which
-are detailed later in this section.
+in one of three ways:
+
+1. Memory Barrier Command (via `vkCmdPipelineBarrier`)
+1. Render Pass specification
+1. Render Pass subpass specification
+
+The memory barrier command is an explicit layout transition command
+that you place in a command buffer.
+For example, you might place one in a command buffer to convert a buffer from
+optimal to linear after executing a `vkCmdCopyBuffer` command
```

Comment:
This seems a bit confused about what resources have layouts, too -- buffers have neither tiling nor layout; only their usage mask provided at creation time.